scripts/notify.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+BIND_ADDRESS=$(influxd print-config --key-name http-bind-address)
+HOST=${BIND_ADDRESS%%:*}
+HOST=${HOST:-"localhost"}
+PORT=${BIND_ADDRESS##*:}
+
+set +e
+result=$(curl -s -o /dev/null http://$HOST:$PORT/ready -w %{http_code})
+while [ "$result" != "200" ]; do
+  sleep 1
+  result=$(curl -s -o /dev/null http://$HOST:$PORT/ready -w %{http_code})
```

Comment:
Do we want to have some sort of escape-hatch here? Like a max number of retries

Reply:
I was wondering about that, but I feel like hitting the sweet spot for that could be a moving target. Also, I believe that `Restart=on-failure` in the service file means that systemd will attempt to restart it anyway. Thoughts?

Reply:
Let's keep it simple for now, we can change it later if people hit problems